cli/index.ts

```diff
@@ -1,47 +1,45 @@
-import { Command, Option } from 'commander';
 import { StatsGroup, StatsMetric } from 'common/types/stats';
-import { registerCommand as registerPreviewCreateCommand } from 'cli/commands/preview/create';
-import { registerCommand as registerPreviewDeleteCommand } from 'cli/commands/preview/delete';
-import { registerCommand as registerPreviewListCommand } from 'cli/commands/preview/list';
-import { registerCommand as registerPreviewUpdateCommand } from 'cli/commands/preview/update';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { registerCommand as registerCreateCommand } from 'cli/commands/preview/create';
+import { registerCommand as registerDeleteCommand } from 'cli/commands/preview/delete';
+import { registerCommand as registerListCommand } from 'cli/commands/preview/list';
+import { registerCommand as registerUpdateCommand } from 'cli/commands/preview/update';
 import { loadTranslations } from 'cli/lib/i18n';
 import { bumpAggregatedUniqueStat } from 'cli/lib/stats';
 import { version } from 'cli/package.json';
+import { OutputFormat, StudioArgv } from 'cli/types';
 
 async function main() {
-	await loadTranslations();
+	const locale = await loadTranslations();
 
-	const studioCommand = new Command();
-
-	studioCommand
-		.name( 'studio' )
-		.description( __( 'Studio by WordPress.com CLI' ) )
+	const studioArgv: StudioArgv = yargs( hideBin( process.argv ) )
+		.scriptName( 'studio' )
+		.usage( __( 'Studio by WordPress.com CLI' ) )
+		.locale( locale )
 		.version( version )
-		.hook( 'preAction', () => {
-			bumpAggregatedUniqueStat( StatsGroup.STUDIO_CLI_USAGE_UNIQUE, StatsMetric.SUCCESS, 'weekly' );
+		.middleware( () =>
+			bumpAggregatedUniqueStat( StatsGroup.STUDIO_CLI_USAGE_UNIQUE, StatsMetric.SUCCESS, 'weekly' )
+		)
+		.option( 'output-format', {
+			type: 'string',
+			hidden: true,
+			coerce: ( value: string ): OutputFormat => {
+				if ( value !== 'json' ) {
+					throw new Error( __( 'The only custom output format supported is "json"' ) );
+				}
+				return value;
+			},
 		} )
-		.addOption(
-			new Option( '--output-format [format]', __( 'Specify a non-standard output format' ) )
-				.argParser( ( value: string ) => {
-					if ( value !== 'json' ) {
-						throw new Error( __( 'The only custom output format supported is "json"' ) );
-					}
-					return value;
-				} )
-				.hideHelp()
-		);
-
-	const previewCommand = studioCommand
-		.command( 'preview' )
-		.description( __( 'Manage preview sites' ) );
-
-	registerPreviewCreateCommand( studioCommand );
-	registerPreviewListCommand( previewCommand, studioCommand );
-	registerPreviewDeleteCommand( previewCommand, studioCommand );
-	registerPreviewUpdateCommand( previewCommand, studioCommand );
+		.command( 'preview', 'Preview commands', ( previewYargs ) => {
+			registerCreateCommand( previewYargs );
+			registerListCommand( previewYargs );
+			registerDeleteCommand( previewYargs );
+			registerUpdateCommand( previewYargs );
+		} );
```

Comment:
```suggestion
		} )
		.demandCommand( 1, __( 'You must provide a valid command' ) )
		.strict();
```
`demandCommand` needed to display the help page by default and `strict` seems like good practice.